src/index.html

```diff
@@ -1,13 +1,74 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="reset.css"
+    >
+    <link
+      rel="stylesheet"
+      href="style.css"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+  </head>
+  <body class="page">
+    <div
+      class="card page__card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        class="card__picture"
+        alt=""
+      >
+      </img>
+      <div class="card__description">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="card__product-code">
+        Код товара: 195434
+      </div>
+      <div class="card__feedback">
+        <div class="card__rating stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__reviews">Отзывов: 5</div>
+      </div>
+      <div class="card__price">
+        <div class="card__price-text">Цена:</div>
+        <div class="card__price-amount">69 999 грн</div>
+      </div>
+      <a
+        href="./"
+        class="card__button"
+        data-qa="hover"
+      >
+        КУПИТЬ
```

Comment:
используй text-transform: uppercase
```suggestion
        Купить
```